### PR TITLE
test cases of workspace and setting

### PR DESCRIPTION
**Pull request by ayushicopods**

### CODE REVIEW

File: cypress/e2e/group-by-test/groupByTestTable.cy.ts [hunk 1]
```diff
@@ -169,7 +169,7 @@ describe("Test for GroupByTestTable, Result", () => {
   })
 
   it("To check elemnts inside perticular test", () => {
-    cy.viewport(1500, 1000)
+    cy.viewport(2000, 1000)
```
Comment (yash6102000) [inline cypress/e2e/group-by-test/groupByTestTable.cy.ts:172]: is that any reason to change the viewport?

File: cypress/e2e/settings-tests/Settings.cy.ts [hunk 2]
```diff
@@ -38,10 +39,10 @@ describe("Test for settings", () => {
     // To check location and title
     getSettings().should("have.text", settings).click()
     cy.location("pathname").should("include", "/settings/general")
-    cy.get("h1").should("have.text", settings)
+    cy.get("h2").should("have.text", settings)
```
Comment (yash6102000) [inline cypress/e2e/settings-tests/Settings.cy.ts:42]: make a function for cy.get("h2")

File: cypress/e2e/settings-tests/Settings.cy.ts [hunk 3]
```diff
@@ -71,7 +72,7 @@ describe("Test for settings", () => {
     getWorkspaces().within(() => {
       cy.get("hr").should(
```
Comment (yash6102000) [inline cypress/e2e/settings-tests/Settings.cy.ts:73]: create function for hr in cypress function file or use it if already available

File: cypress/e2e/workspace-tests/Workspace.cy.ts [hunk 4]
```diff
@@ -26,7 +26,7 @@ describe("Test for settings", () => {
     // To check location and title
     getSettings().should("have.text", settings).click()
     cy.location("pathname").should("include", "/settings/general")
-    cy.get("h1").should("have.text", settings)
+    cy.get("h2").should("have.text", settings)
```
Comment (yash6102000) [inline cypress/e2e/workspace-tests/Workspace.cy.ts:29]: same as above

File: cypress/e2e/workspace-tests/Workspace.cy.ts [hunk 5]
```diff
@@ -36,7 +36,7 @@ describe("Test for settings", () => {
     getWorkspaces().within(() => {
       cy.get("hr").should(
```
Comment (yash6102000) [inline cypress/e2e/workspace-tests/Workspace.cy.ts:37]: same as above

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**